### DISCUSSION TITLE
after upload error have a loop since a day in the upload jobq

### DISCUSSION THREAD

**Discussion by ferenckumi** [LycheeOrg/Lychee#2395] [Bugs] [answered]
### Lychee version

v5.1.2

### Which PHP version are you using?

PHP 8.3

### Detailed description of the problem

after a new setup lychee 5.1.2 get an error in relation big a foto. I change then the setup for imagic and load the photo again with success...  

Titel
IMG_1706_IMG_1839.jpg
Hochgeladen
Apr 19, 2024, 5:32:46 PM Europe/Zurich
Bild
Größe
162.85 MB
Format
image/jpeg
Auflösung
### 25991 x 7428
Kamera
Aufgenommen
Jan 18, 2024, 3:18:23 PM -05:00
Marke
Apple
Typ/Modell
iPhone 15 Pro Max
Objektiv
iPhone 15 Pro Max back triple camera 6.86mm f/1.78
Verschlusszeit
1/7937 sec
Blende
ƒ / 1.8
Brennweite
7 mm
ISO
ISO 80
<img width="499" alt="Bildschirmfoto 2024-04-20 um 07 30 39" src="https://github.com/LycheeOrg/Lychee/assets/18476970/1fee9bdc-3cda-4846-954e-d9fc077b23f3">

since over 1 day a have 2 uploads in the JOBQ an will not finished.  

how can I clean the jobq , thanks 

### Steps to reproduce the issue

1 set imagick resolution of a lover value 
2 try load a bigger photo

... maybe some relation to reload php and appache

### Diagnostics [REQUIRED]

    Diagnostics
    -----------
        Warning: Dropbox import not working. dropbox_key is empty.
    
    System Information
    ------------------
        Lychee Version (git):                    master (e534bc8) -- More than 30 commits behind (16 hours ago).
        DB Version:                              5.1.2
        
        composer install:                        dev
        APP_ENV:                                 production
        APP_DEBUG:                               false
        APP_URL:                                 set
        APP_DIR:                                 default
        LOG_VIEWER_ENABLED:                      true
        LIVEWIRE_ENABLED:                        true
        
        System:                                  Linux
        PHP Version:                             8.3.6
        PHP User agent:                          Lychee/4 (https://lycheeorg.github.io/)
        Timezone:                                Europe/Zurich
        Max uploaded file size:                  500M
        Max post size:                           500M
        Livewire chunk size:                     12.00 MB
        Max execution time:                      200
        MySQL Version:                           8.0.36-0ubuntu0.22.04.1
        
        exec() Available:                        yes
        Imagick Available:                       1
        Imagick Enabled:                         1
        Imagick Version:                         1691
        GD Version:                              2.3.3
        Number of foreign key:                   12 found.
    
    Config Information
    ------------------
        version:                                 050102
        check_for_updates:                       1
        sorting_photos_col:                      title
        sorting_photos_order:                    ASC
        sorting_albums_col:                      title
        sorting_albums_order:                    ASC
        imagick:                                 1
        skip_duplicates:                         0
        small_max_width:                         0
        small_max_height:                        360
        medium_max_width:                        1920
        medium_max_height:                       1080
        lang:                                    de
        image_overlay_type:                      desc
        default_license:                         none
        compression_quality:                     90
        grants_full_photo_access:                1
        delete_imported:                         0
        mod_frame_enabled:                       1
        mod_frame_refresh:                       30
        thumb_2x:                                1
        small_2x:                                1
        medium_2x:                               1
        landing_page_enable:                     0
        site_owner:                              ------------
        landing_title:                           John Smith
        landing_subtitle:                        Cats, Dogs & Humans Photography
        sm_facebook_url:                         https://www.facebook.com/JohnSmith
        sm_flickr_url:                           https://www.flickr.com/JohnSmith
        sm_twitter_url:                          https://www.twitter.com/JohnSmith
        sm_instagram_url:                        https://instagram.com/JohnSmith
        sm_youtube_url:                          https://www.youtube.com/JohnSmith
        landing_background:                      dist/cat.webp
        site_title:                              Lychee v5
        footer_show_copyright:                   1
        site_copyright_begin:                    2019
        site_copyright_end:                      2024
        footer_additional_text:                  
        footer_show_social_media:                0
        search_public:                           0
        SL_enable:                               0
        SL_for_admin:                            0
        recent_age:                              1
        grants_download:                         0
        photos_wraparound:                       1
        map_display:                             0
        zip64:                                   1
        map_display_public:                      0
        map_provider:                            Wikimedia
        force_32bit_ids:                         0
        map_include_subalbums:                   0
        update_check_every_days:                 3
        has_exiftool:                            0
        share_button_visible:                    0
        import_via_symlink:                      0
        has_ffmpeg:                              0
        location_decoding:                       0
        location_decoding_timeout:               30
        location_show:                           1
        location_show_public:                    0
        rss_enable:                              0
        rss_recent_days:                         7
        rss_max_items:                           100
        prefer_available_xmp_metadata:           0
        editor_enabled:                          1
        lossless_optimization:                   0
        swipe_tolerance_x:                       150
        swipe_tolerance_y:                       250
        local_takestamp_video_formats:           .avi|.mov
        log_max_num_line:                        1000
        unlock_password_photos_with_url_param:   0
        nsfw_visible:                            1
        nsfw_blur:                               0
        nsfw_warning:                            0
        nsfw_warning_admin:                      0
        nsfw_banner_override:                    
        map_display_direction:                   1
        album_subtitle_type:                     oldstyle
        upload_processing_limit:                 4
        new_photos_notification:                 0
        legacy_id_redirection:                   1
        zip_deflate_level:                       6
        SA_enabled:                              1
        default_album_protection:                1
        album_decoration:                        layers
        album_decoration_orientation:            row
        allow_username_change:                   1
        auto_fix_orientation:                    1
        use_job_queues:                          0
        random_album_id:                         starred
        use_last_modified_date_when_no_exif_date: 0
        ffmpeg_path:                             
        ffprobe_path:                            
        layout:                                  justified
        date_format_photo_thumb:                 M j, Y, g:i:s A e
        date_format_photo_overlay:               M j, Y, g:i:s A e
        date_format_sidebar_uploaded:            M j, Y, g:i:s A e
        date_format_sidebar_taken_at:            M j, Y, g:i:s A e
        date_format_hero_min_max:                F Y
        date_format_hero_created_at:             M j, Y, g:i:s A T
        date_format_album_thumb:                 M Y
        upload_chunk_size:                       0
        nsfw_banner_blur_backdrop:               0
        search_pagination_limit:                 1000
        search_minimum_length_required:          4
        photo_layout_justified_row_height:       320
        photo_layout_masonry_column_width:       300
        photo_layout_grid_column_width:          250
        photo_layout_square_column_width:        200
        photo_layout_gap:                        12
        display_thumb_album_overlay:             always
        display_thumb_photo_overlay:             hover
        default_album_thumb_aspect_ratio:        1/1
        use_album_compact_header:                0
        login_button_position:                   left
        back_button_enabled:                     0
        back_button_text:                        Return to Home
        back_button_url:                         /
        current_job_processing_visible:          1

### Browser & System [REQUIRED]

I load from a 

mecbook 
sonoma 14.4.1

browser 
Brave
v 1.64.122

Lychee 
server 

<img width="579" alt="Bildschirmfoto 2024-04-20 um 07 52 48" src="https://github.com/LycheeOrg/Lychee/assets/18476970/22c90922-4dae-47bf-90ec-da1206f18d5b">

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment, marked as answer]
> how can I clean the jobq , thanks

Go to Left menu => Maintenance. There will be a module waiting for you. :)
